rmgpy/molecule/pathfinderTest.py

```diff
@@ -312,18 +312,14 @@ def test_c5h6o2(self):
         expected_idx_path = [3, 2, 4, 6]
         self.assertEquals(idx_path, expected_idx_path)
 
-    @work_in_progress
     def test_c6h6o4(self):
-        # Can't read this InChI:
-        # the old OpenBabel could read it, but the resulting molecule had an invalid adjacency_list
-        inchi = "InChI=1S/C6H6O4/c1-2-4-9-6(7)3-5-10-8/h2-3H,1,5H2"
+        inchi = "InChI=1S/C8H14O4S/c1-3-6-13(2,11)7-8(9)4-5-12-10/h3,6H,1,4-5,7H2,2H3,(H-,10,11)"
```

Comment:
One suggestion, can we change the name of the test? (since it's no longer c6h6o4) and perhaps add a docstring or comment so future people know what we're trying to test (which would have helped previously).

Reply:
I amended the commit when rebasing and changed the name of the test.